File: src/main/scala/com/github/mauricio/postgresql/exceptions/EncoderNotAvailableException.scala
package com.github.mauricio.postgresql.exceptions

import com.github.mauricio.postgresql.messages.frontend.FrontendMessage

/**
 * User: Maurício Linhares
 * Date: 3/4/12
 * Time: 12:19 AM
 */

class EncoderNotAvailableException( message : FrontendMessage )
  extends IllegalArgumentException( "Encoder not available for name %s".format( message.kind ) )

File: db-async-common/src/main/java/com/github/jasync/sql/db/exceptions/EncoderNotAvailableException.kt
package com.github.jasync.sql.db.exceptions

import com.github.jasync.sql.db.KindedMessage

@Suppress("RedundantVisibilityModifier")
public class EncoderNotAvailableException(message: KindedMessage) :
    DatabaseException("Encoder not available for name %s".format(message.kind))
